docs/source/workshop.rst

```diff
@@ -1,4 +1,361 @@
+	 'writers': ['ICML.cc/2019']}
+
+Please note that this conference group does not show up under the header "Open for Submissions" on Openreview homepage until an invitation for submission is created.
```

Comment:
We can add that the home page of the conference is available through the link: https://dev.openreview.net/group?id=ICML.cc/2019/Confernece, this page renders the content of the webfield.